app/views/runners/index.html.erb

```diff
@@ -16,30 +16,30 @@
     <strong>ROLE</strong>
   </div>
   <% if runner_role == 'Coach' %>
-  <div class="col-md-4 col-sm-4 col-xs-5">
+  <div class="col-md-4 col-sm-4 col-xs-4">
     <strong>OPTIONS</strong>
   </div>
   <% end %>
 </div>
 
 <% @runners.each.with_index do |runner, index| %>
-  <% if index % 2 ==0 %>
+  <% if index % 2 == 0 %>
```

Comment:
Hmm so there is a bootstrap "striped" table here: http://getbootstrap.com/css/#tables-striped but that would be in use with tables which I've decided to not use. It is possible to do with css but it starts getting into some intense css. Here is a fiddle of a potential way: http://jsfiddle.net/aniskhan001/32KEX/ I'd honestly like to know what you would do. Shall I "css-atize" it or should I keep it as logic on the view pages?